go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/gofrs/flock
+
+go 1.21.0
```

Comment:
Would have been nice to avoid the patch in go version as now anyone depending on this module needs to set `go 1.21.0` and can't just use `go 1.21`.

Could we put `go 1.21` instead?